webapp/src/gitjson.tsx

```diff
@@ -235,8 +235,8 @@ class GithubComponent extends data.Component<GithubProps, GithubState> {
             }
         }))
 
-        // only branch from master...
-        if (gid.tag == "master") {
+        // only branch from main and master...
+        if (gid.tag === "master" || gid.tag === "main") {
```

Comment:
There is a _default_branch_ in github repo which you should query and do this?

Reply:
_default_branch_ is configurable and need not be main.

Reply:
Feel free to be not blocked on this.

Reply:
Yeah this is not a fool-proof fix by any means. It's just a targeted change to unblock a bit of functionality. I think making us more resilient to different default branches would be nice, but a larger change.

Reply:
Filed this as a follow-up, though I expect it's somewhat low-pri: https://github.com/microsoft/pxt/issues/9744